lib/cn/cn_tree.c

```diff
@@ -3453,6 +3461,25 @@ cn_tree_perfc_shape_report(
     }
 }
 
+enum hse_mclass
+cn_tree_node_mclass(struct cn_tree_node *tn, enum hse_mclass_policy_dtype dtype)
+{
+    struct mclass_policy *policy;
+    enum hse_mclass_policy_age age;
+
+    INVARIANT(tn);
+
+    /* HSE-REVISIT: This check exist only to satisfy unit test. Needs to be removed */
+    if (!tn->tn_tree || !tn->tn_tree->cn)
```

Comment:
Can this not be mocked?

Reply:
Couldn't figure out how to mock this function and gave up.. There are callers to this function in the same file as the definition, so no mock substitution happens. Any thoughts on how to get around this issue?

Reply:
I would make it a weak symbol and redefine in the test source file. I think that should be enough.

Reply:
Sweet, HSE_WEAK worked, thanks :-) Btw, are unit tests part of a separate library?